pkgs/by-name/up/upgrade-assistant/package.nix

```
@@ -0,0 +1,16 @@
+{ lib, buildDotnetGlobalTool, ... }:
+buildDotnetGlobalTool {
+  pname = "upgrade-assistant";
+  version = "0.5.820";
+
+  nugetHash = "sha256-GB+q5aZRkBTeXUbIPjkPsll6pSI/H6Iyh5mY53uT284=";
+
+  meta = {
+    homepage = "https://github.com/dotnet/upgrade-assistant";
+    description = "A tool to assist developers in upgrading .NET Framework applications to .NET 6 and beyond";
```

```suggestion
    description = "Tool to assist developers in upgrading .NET Framework applications to .NET 6 and beyond";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes